services/company_budget/app/src/Domain/ValueObject/BudgetName.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Domain\ValueObject;
+
+class BudgetName {
+  private string $name;
+  
+  public function __construct(string $name) {
+    $this->name = $name;
+    $this->ensureNameOnlyContainsLetters($name);
+  }
+
+  private function ensureNameOnlyContainsLetters(string $value): void {
```

Comment:
### Question
Will budget names have only letters?
What if I want to use "My budget 1" and "My budget 2" as names?